test/indexing/test_getitem.py

```diff
@@ -335,6 +335,56 @@ def test_index_has_range(self):
         np.testing.assert_allclose(res[0], np_res)
 
 
+class TestGetitemBasicIndexOutputView(unittest.TestCase):
+    def setUp(self):
+        # Stride now only supports in dygraph mode
+        paddle.disable_static()
+
+    def test_index_is_int(self):
+        np_data = np.ones((5, 5, 5), dtype='float32')
+        np_tmp = np_data[3, 2]
+        np_tmp[2] = 20
+
+        x = paddle.ones((5, 5, 5), dtype='float32')
+        x_tmp = x[3, 2]
+        x_tmp[2] = 20
+
+        np.testing.assert_allclose(x.numpy(), np_data)
+
+    def test_index_is_0dTensor(self):
+        np_data = np.ones((5, 5, 5), dtype='float32')
+        np_tmp = np_data[3, 2]
+        np_tmp[2] = 20
+
+        x = paddle.ones((5, 5, 5), dtype='float32')
+        x_tmp = x[paddle.to_tensor(3), paddle.to_tensor(2)]
+        x_tmp[2] = 20
+
+        np.testing.assert_allclose(x.numpy(), np_data)
+
+    def test_index_is_slice(self):
+        np_data = np.ones((5, 5, 5), dtype='float32')
+        np_tmp = np_data[::2, :, 0:4]
+        np_tmp[2] = 20
+
+        x = paddle.ones((5, 5, 5), dtype='float32')
+        x_tmp = x[::2, :, 0:4]
+        x_tmp[2] = 20
+
+        np.testing.assert_allclose(x.numpy(), np_data)
+
+    def test_index_is_None(self):
+        np_data = np.ones((5, 5, 5), dtype='float32')
+        np_tmp = np_data[None]
+        np_tmp[:, 2] = 20
+
+        x = paddle.ones((5, 5, 5), dtype='float32')
+        x_tmp = x[None]
+        x_tmp[:, 2] = 20
+
+        np.testing.assert_allclose(x.numpy(), np_data)
+
```

Comment:
shall we add test case of ellipsis index

Reply:
done, add a case to test ellipsis index.